Source/AtlusScriptLibrary/Common/Libraries/FlowScriptModuleFunction.cs

```diff
@@ -20,8 +20,8 @@ public class FlowScriptModuleFunction : ICloneable
 
         public string Description { get; set; }
 
-    [JsonConverter(typeof(HexIntStringJsonConverter))]
-    public int Address { get; set; }
+    [JsonConverter(typeof(HexUIntStringJsonConverter))]
+    public uint Address { get; set; }
```

Comment:
Nit: should probably be ulong, considering it's used by 64-bit games as well..

Reply:
done